testCommon/src/test/resources/setup-with-single-remote.sh

```diff
@@ -22,6 +22,11 @@ cd machete-sandbox
     # Let's clean up the upstream so that we can't rely on `git config` for finding out the remote branch;
     # we'll need to match the local branch to its remote tracking branch by name.
     git branch --unset-upstream
+  create_branch update-icons
```

Comment:
additional branch - to perform current/noncurrent branch **pull** (preparation for mergin ui test cases #843)